packages/spindle-icons/scripts/icon-doc.ts

```diff
@@ -22,7 +25,7 @@ const createIconList = async (): Promise<void> => {
     const icons = files
       .filter((file) => file.endsWith('.svg'))
       .filter((file) => !file.startsWith('sprite'));
-    fs.writeFile('icons.md', markdown(icons));
+    fs.writeFile(`${DOC_DIR}/icons.md`, markdown(icons));
```

Comment:
fs.promisesをfsとして使ってるのでawaitをつけた方がいいのかなあと思いました。

```suggestion
    await fs.writeFile(`${DOC_DIR}/icons.md`, markdown(icons));
```

Reply:
漏れてましたーありがとうございます:pray: